src/Microsoft.XmlSerializer.Generator/dir.props

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <PackageVersion>1.0.0</PackageVersion>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
This should not set IsNETCoreApp=true. In fact you should just remove all these Is* properties from this project.